ci-operator/jobs/openshift/elasticsearch-operator/openshift-elasticsearch-operator-feature-es6x-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/elasticsearch-operator:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
this makes it so we have to manually invoke this test... is this what we want?

Reply:
yeah we're basically disabling this test and replacing it with the "large worker" version below.

Reply:
> this makes it so we have to manually invoke this test... is this what we want?

I don't recall but this job is duplicated by another and the only difference is we configure the worker nodes to be larger so that we get more resources.